### ISSUE TITLE
Ubuntu 18.04.1 LTS apt install documentation

### ISSUE THREAD

**Issue by pndaly** [iraf-community/iraf#71]
Hi,

I read that IRAF was now part of the Ubuntu 18 distribution. Certainly, the command:

 % apt install iraf iraf-dev iraf-noao iraf-noao-dev iraf-wcstools iraf-rvsao

seems to complete and things appear to be installed in /usr/bin/iraf. An icon is created but right-clicking on that and selecting "Show details" returns "No Application Found". The install in /usr/lib/iraf also looks minimal and I cannot find 'mkiraf' to continue. It does seem to install a version of pyraf that appears to work. So, I'm curious as to how to proceed. I'd like the command line version (via 'cl') to work too but can't seem to get there. (I did find a mkiraf.sh deeply buried in /usr/lib/iraf but it doesn't run as it has some hard-coded paths
in there that do not exist eg iraf='/build/iraf-wlsLkZ/iraf-2.16.1+2018.03.10').

Also, "apt install iraf-fitsutil iraf-mscred iraf-sptable" returns 'E: unable to locate package' so something is not quite right there.

I can test out any fixes/suggestions you have. I'd like to get this to work as it's a lot simpler than a manual IRAF install.

Thanks for all you hard work.

**Comment by olebole** [comment]
The command name to start IRAF had to be changed, due to a conflict with Common List. So, on Ubuntu (and Debian), the command is `irafcl` for the command line, resp. `irafcl -x` for the xterm (if installed). Just clicking on the icon should also run the xterm based one -- maybe you need `sudo apt install xterm` before.
`mkiraf` is not needed anymore. The hard-coded build paths there are a bug, however; I'll fix the script here.
Does this help? If not, please report back.

fitsutil, mscred and sptable are packages that were created only after the release of Ubuntu 18.04, so they are not available there. To get them into Ubuntu 18.04, a ["backport"](https://help.ubuntu.com/community/UbuntuBackports) would be needed. Although this is not too complicated (see [Ubuntu Backports](https://wiki.ubuntu.com/UbuntuBackports)), I don't want to get into this myself. So, if you want to help, you are welcome to contribute ;-) I would however provide any help you need there, since Backports currently if one of the big unresolved questions in [Debian Astro](https://blends.debian.org/astro) (where the IRAF packages in Ubuntu originate from).

**Comment by olebole** [comment]
If you find any error, please report. For the external packages, I don't know which are important, so if something is missing please tell me. However, some packages can't included due to licensing problems; for example the STSDAS package. For most other packages (including those from NOAO), the license is not really clear; a clarification that they follow the main IRAF license would be nice here.

**Comment by olebole** [comment]
@pndaly is this fixed now? Then, I would close the issue.

**Comment by olebole** [comment]
Please reopen or file a new issue if there are still problems.
